templates/components/panels/get-involved.hbs

```diff
@@ -4,20 +4,20 @@
       <h2>Get involved</h2>
       <div class="highlight"></div>
     </header>
-    <div class="row media">
-      <div class="six columns" id="read-rust">
+    <div class="flex flex-column flex-row-l">
+      <div id="read-rust" class="mw-50-l mh2-l pt0 flex flex-column justify-between-l">
```

Comment:
```suggestion
      <div id="read-rust" class="mw-50-l mr4-l pt0 flex flex-column justify-between-l">
```

Using `mh2-l` you also add margin on the left, removing the alignment with the row below:

![Istantanea_2019-05-15_19-23-18](https://user-images.githubusercontent.com/2299951/57795715-f4ccf680-7746-11e9-87ef-575afd790118.png)

If you instead add a `mr4-l` on the left column and no margin on the right one the content will be properly aligned.